OpenTap.Metrics/MetricManager.cs

```diff
@@ -83,7 +89,15 @@ public static IEnumerable<MetricInfo> GetMetricInfos()
             foreach (var member in memberGrp)
             {
                 foreach (var mem in member)
+                {
+                    if (_pushMetricInfos.TryGetValue(mem, out var existingMetricInfo))
+                    {
+                        yield return existingMetricInfo;
+                        continue;
+                    }
+
                     yield return new MetricInfo(mem, member.Key, metricSource);
```

Comment:
I think a regular else would be more clear here than this `if (...) continue` pattern in this case.

Reply:
Sure, changed.